src/AccessControl/RouteAccessControlFactory.php

```diff
@@ -76,8 +76,9 @@ protected function getAccessControlExpressionForApiPlatform(Route $route): strin
             try {
                 $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
                 $attributes = AttributesExtractor::extractAttributes($route->getDefaults());
-                $isGranted = $resourceMetadata->getOperationAttribute($attributes, 'security', null, true);
```

Comment:
`isGranted` is returned by the function, so now you will return `RouteAccessControlData::RESOURCE_UNRELATED_ROUTE` instead of `security` or `security_post_denormalize`.
Can you make sure the correct value is returned?
Great improvement besides! 🙏